eq3bt/connection.py

```diff
@@ -76,6 +77,5 @@ def make_request(self, handle, value, timeout=DEFAULT_TIMEOUT, with_response=Tru
                     _LOGGER.debug("Waiting for notifications for %s", timeout)
                     self._conn.waitForNotifications(timeout)
         except btle.BTLEException as ex:
-            _LOGGER.debug("Got exception from bluepy while making a request: %s", ex)
+            _LOGGER.info("Got exception from bluepy while making a request: %s", ex)
```

Comment:
```suggestion
            _LOGGER.debug("Got exception from bluepy while making a request: %s", ex)
```
Let's avoid info level debugging here, too.

Reply:
That's done, see af880ea